lib/chef/client.rb

```diff
@@ -323,6 +325,22 @@ def run
     # @todo make this stuff protected or private
     #
 
+    # @api private
+    def warn_if_eol
+      require_relative "version"
+
+      # we have a yearly release and we know 15 goes EOL in 2021 so calculate off that
```

Comment:
```suggestion
      # we have a yearly release and we know 15 goes EOL in 2022 so calculate off that
```

Is this comment wrong?

Reply:
isn't that comment right?  15 expires next year.

Reply:
One of the tests is testing that 15 gets an EOL message mentioning 2022. Is the comment right or is the test? Or am I just very confused?

Reply:
I set the dates in the specs to be right on the border now so it should line up better with the dates.